README.md

```diff
@@ -12,11 +12,11 @@
 
 A node client for the Customer.io [REST API](https://customer.io/docs/api/).
 
-## React Native and alternative Node runtimes
+## Alternative Node runtimes
```

Comment:
Should be link to our react native and expo apis here?

Reply:
Maybe we should: https://customer.io/docs/sdk/react-native/getting-started/?